data.sql

```diff
@@ -1,5 +1,14 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('Agumon', date '2020-02-03', 0, true, '10.23');
+
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('Gabumon', date '2018-11-15', 2, true, '8');
+
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('Pikachu', date '2021-01-07', 0, false, '15.04');
+
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES ('De', date '2017-05-12', 2, true, '11');
```

Comment:
- You have inputted the wrong information for this data, please fix this, according to the project requirements it should be:
```sql
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Devimon', '2017/05/12', '5', 'true', '11.0');
```